src/mlpack/methods/bayesian_linear_regression/bayesian_linear_regression_main.cpp

```diff
@@ -0,0 +1,195 @@
+    "at all, a model can be passed via the " +
+    PRINT_PARAM_STRING("input_model") + " parameter."
+    "\n\n"
+    "The program can also provide predictions for test data using either the "
+    "trained model or the given input model.  Test points can be specified "
+    "with the " + PRINT_PARAM_STRING("test") + " parameter.  Predicted "
+    "responses to the test points can be saved with the " +
+    PRINT_PARAM_STRING("predictions") + " output parameter. The "
+    "corresponding standard deviation can be save by precising the " +
+    PRINT_PARAM_STRING("stds") + " parameter."
+    "\n\n"
+    "For example, the following command trains a model on the data " +
+    PRINT_DATASET("data") + " and responses " + PRINT_DATASET("responses") +
+    "with center set to true and scale set to false (so, Bayesian "
+    "linear regression is being solved, and then the model is saved to " +
+    PRINT_MODEL("bayesian_linear_regression_model") + ":"
+    "\n\n" +
+    PRINT_CALL("bayesian_linear_regression", "input", "data", "responses",
+               "responses", "center", 1, "scale", 0, "output_model",
+               "bayesian_linear_regression_model") +
+    "\n\n"
+    "The following command uses the " +
+    PRINT_MODEL("bayesian_linear_regression_model") + " to provide predicted " +
+    " responses for the data " + PRINT_DATASET("test") + " and save those " +
+    " responses to " + PRINT_DATASET("test_predictions") + ": "
+    "\n\n" +
+    PRINT_CALL("bayesian_linear_regression", "input_model",
+               "bayesian_linear_regression_model", "test", "test",
+               "predictions", "test_predictions") +
+    "\n\n"
+    "Because the estimator computes a predictive distribution instead of "
+    "simple point estimate, the " + PRINT_PARAM_STRING("stds") + " parameter "
+    "allows to save the prediction uncertainties with one standard deviation "
+    "from the mean :"
+    "\n\n" +
+    PRINT_CALL("bayesian_linear_regression", "input_model",
+               "bayesian_linear_regression_model", "test", "test",
+               "predictions", "test_predictions", "stds", "stds"));
+
+PARAM_MATRIX_IN("input", "Matrix of covariates (X).", "i");
+
+PARAM_ROW_IN("responses", "Matrix of responses/observations (y).", "r");
+
+PARAM_MODEL_IN(BayesianLinearRegression, "input_model", "Trained "
+               "BayesianLinearRegression model to use.", "m");
+
+PARAM_MODEL_OUT(BayesianLinearRegression, "output_model", "Output "
+                "BayesianLinearRegression model.", "M");
+
+PARAM_MATRIX_IN("test", "Matrix containing points to regress on (test "
+                "points).", "t");
+
+PARAM_MATRIX_OUT("predictions", "If --test_file is specified, this "
+                  "file is where the predicted responses will be saved.", "o");
+
+PARAM_MATRIX_OUT("stds", "If --std_file is specified, this file is where "
+                 "the standard deviations of the predictive distribution will "
+                 "be saved.", "u");
```

Comment:
```suggestion
PARAM_MATRIX_OUT("stds", "If specified, this is where the standard deviations "
    "of the predictive distribution will be saved.", "u");
```
I modified this to not be specific to command-line bindings; the same text will be displayed in Python, Julia, Go, and any other languages, so I tried to avoid the use of the term `file` since it doesn't apply in those languages. :)